.storybook/preview.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import {addParameters, addDecorator} from '@storybook/react';
 import {setConsoleOptions} from '@storybook/addon-console';
 import {withContexts} from '@storybook/addon-contexts/react';
+import {color, withKnobs} from '@storybook/addon-knobs';
+import DefaultThemeColors from '@shopify/polaris-tokens/dist/base.json';
```

Comment:
Gonna need a swift rebase after https://github.com/Shopify/polaris-react/pull/2763 as the path to this has changed slightly (`@shopify/polaris-tokens/dist-modern/themes/base.json` off the top of my head)

Reply:
Thanks, just did it!